homeassistant/components/rest/switch.py

```diff
@@ -71,6 +75,7 @@
         vol.Inclusive(CONF_USERNAME, "authentication"): cv.string,
         vol.Inclusive(CONF_PASSWORD, "authentication"): cv.string,
         vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
+        vol.Optional(CONF_AVAILABILITY): cv.template,
```

Comment:
Is this a new configuration option, if so, we should document it in a documentation PR?

Reply:
Yes, will add doc PR